features/shop/cart_with_single_tax_rate.feature

```diff
@@ -0,0 +1,36 @@
+@ui-cart
+Feature: Cart with single tax rate
+    In order to buy goods with correct taxes applied
```

Comment:
Why? This feature is about taxes. So the main benefit it is proper taxes. Otherwise we could say ``correct total price`` for everything cart related, not very helpful.